imageio/plugins/spe.py

```diff
@@ -387,6 +394,12 @@ def _parse_header(self, spec):
                 ret[name] = v
             return ret
 
+        def _init_meta_data_post_v3(self):
+            info = self._parse_header(Spec.basic)
+            self._file.seek(info["xml_footer_offset"])
+            xml = self._file.read()
+            self._meta = {"__xml": xml}
+
```

Comment:
This function looks much shorter than the `pre_v3` version. Does it mean that when we merge this PR, the meta dict will look different for some files? Some users may depend on the meta data structure.

Reply:
Yes, but the point is that I don't have the old metadata entries available (I would need to understand the xml schema for that).
Right now they're read from the "old-style" header, but SPE3 files don't have that header filled correctly (this is explicitly allowed by the spec, which says "read the xml if you want the correct metadata"), so the metadata entries there are basically just made up with no warning to the end user, which seems worse.

Reply:
Thanks. So if a user reads an SPE3 file *now*, and inspect its metadata, and does the same after the PR is merged, there won't be anything missing, only added?

Reply:
No; some fields, which were previously filled with nonsense, will be missing after this PR.

Reply:
> which were previously filled with nonsense

Ha! ok, sounds good to me.